### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by NatKarmios**
Just to make PyCharm's inspector happy :P

### CODE REVIEW

File: src/extensions/modtools.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def __init__(self, bot):
         inumber = int(number)
         if inumber <= 100:
             # In order to delete the command message too, the number of messages to clear is incremented
+            # noinspection PyUnresolvedReferences
```
Comment (tom-sherman) [inline src/extensions/modtools.py:56]: I don't get any complaints from pycharm, what are you receiving?

Comment (NatKarmios) [reply src/extensions/modtools.py:56]: A couple of lines too long and some unused imports.

Comment (NatKarmios) [reply src/extensions/modtools.py:56]: Actually, don't bother with this one, about to push with another commit with some more type hinting for the util functions.

Comment (tom-sherman) [reply src/extensions/modtools.py:56]: Re the `PyUnresolvedReferences`?

Comment (NatKarmios) [reply src/extensions/modtools.py:56]: Oh, that one, for some reason PyCharm is complaining about the return types of coroutines.

**Comment by NatKarmios** [comment]
[Here's a screenshot](https://puu.sh/w9Re2/fdcf434ac1.png)

File: src/extensions/modtools.py (hunk 1 above)
Comment (NatKarmios) [reply src/extensions/modtools.py:56]: [Here's a screenshot](https://puu.sh/w9Re2/fdcf434ac1.png)

Comment (tom-sherman) [reply src/extensions/modtools.py:56]: I don't get that warning, weird

Comment (NatKarmios) [reply src/extensions/modtools.py:56]: Yeah, that's why I just suppressed it.
I suppose I should just disable that particular inspection and not bother with the in-comment suppressions.

**Pull request closed without merging**